ansible_collections/serverscom/sc_api/plugins/modules/sc_load_balancer_instance_l7.py

```diff
@@ -0,0 +1,623 @@
+        description: List of port numbers to apply the rule. Must be non-empty.
+        required: true
+        type: list
+        elements: int
+      ssl:
+        description: Enable SSL termination.
+        type: bool
+        default: false
+      http2:
+        description: Enable HTTP/2 protocol.
+        type: bool
+        default: false
+      http_to_https_redirect:
+        description: Redirect HTTP traffic to HTTPS.
+        type: bool
+        default: false
+      http2_push_preload:
+        description: Enable HTTP/2 push preload.
+        type: bool
+        default: false
+      domains:
+        description: List of domain names for the rule. Must be non-empty.
+        required: true
+        type: list
+        elements: str
+      ssl_certificate_id:
+        description: "SSL certificate identifier."
+        type: str
+      location_zones:
+        description: "List of location zones (rewrite rules)."
+        type: list
+        elements: dict
+      real_ip_header:
+        description: Real IP header configuration.
+        type: dict
+  upstream_zones:
+    description: List of upstream zones (groups of upstream servers with predefined parameters).
+    type: list
+    elements: dict
+    suboptions:
+      id:
+        description: Unique identifier of a zone. Must be a string between 1 and 255 characters.
+        required: true
+        type: str
+      method:
+        description: Load balancing method.
+        type: str
+        default: "random.least_conn"
+        choices: ["random.least_conn", "round-robin", "least_conn"]
+      ssl:
+        description: Enable SSL termination.
+        type: bool
+        default: false
+      sticky:
+        description: Enable sticky cookie.
+        type: bool
+        default: true
+      hc_interval:
+        description: "Health check interval in seconds. Range: 1 to 60."
+        type: int
+        default: 5
+      hc_jitter:
+        description: "Health check jitter in seconds. Range: 0 to 60."
+        type: int
+        default: 5
+      hc_fails:
+        description: Number of failed health checks before taking an upstream out of service. Must be at least 1.
+        type: int
+        default: 3
+      hc_passes:
+        description: Number of successful health checks to reinstate an upstream. Must be at least 1.
+        type: int
+        default: 3
+      hc_domain:
+        description: Domain for health check.
+        type: str
+      hc_path:
+        description: Health check path. Must be up to 255 characters.
+        type: str
+        default: "/"
+      hc_method:
+        description: Health check HTTP method.
+        type: str
+        default: "GET"
+        choices: ["GET", "HEAD"]
+      hc_mandatory:
+        description: Enable mandatory health check for new upstream servers.
+        type: bool
+        default: false
+      hc_status:
+        description: Expected health check status. Must be up to 255 characters.
```

Comment:
It has own format, I think, it's better to explain, what it is (e.g. range as start-end, if it's inclusive for last value or not), can you specify a single value or not, list of values separated by coma (is it?).